app/Schedule/View.elm

```diff
@@ -269,6 +274,16 @@ predictionColor minutesLate =
             Color.red
 
 
+arrival : Date -> Train -> Node Msg
+arrival date train =
+    trainMetadata ("arrives at" ++ arrivalTime train)
+
+
+arrivalTime : Train -> String
+arrivalTime train =
+    prettyTime train.scheduledArrival
```

Comment:
I didn't know you could pointfree that way. I guess that's cool. Feels a little obscure, though that feels like a weird thing to say in languages that end up relatively cryptic like this anyway.